docs-markdown/src/helper/metadata.ts

```diff
@@ -0,0 +1,48 @@
+import { updateMetadataDate } from '../controllers/metadata-controller';
+import { workspace, window } from 'vscode';
+import { noActiveEditorMessage, toShortDate, isMarkdownFileCheck } from './common';
+import { findReplacement } from './utility';
+
+const blacklist: string[] = [];
+
+export async function nag() {
+	if (workspace.getConfiguration('markdown').metadataNag === false) {
+		return;
+	}
+	const editor = window.activeTextEditor;
+	if (!editor) {
+		noActiveEditorMessage();
+		return;
+	}
+
+	if (!isMarkdownFileCheck(editor, false)) {
+		return;
+	}
+
+	const fileName = editor.document.fileName;
+	// skip files that have already been saved
+	if (blacklist.indexOf(fileName) > -1 === true) {
```

Comment:
Please reconsider naming this variable to something other than `blacklist`, perhaps `disallowlist`?

Reply:
Hi @IEvangelist , good call. I believe there's a feature/work item to rename all of these instances soon.  Thanks